---
 src/pypass.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/pypass.py b/src/pypass.py
index 60799e2..ba8f7c6 100644
--- a/src/pypass.py
+++ b/src/pypass.py
@@ -2,9 +2,8 @@ import argparse
 from pykeepass import PyKeePass
 parser = argparse.ArgumentParser()
 parser.add_argument("--query", required=True,help="Title of entry which you are looking for.(Case sensitive)")
-parser.add_argument("--password", required=True,help="Password for keepass database.")
 args = parser.parse_args()
-pm = PyKeePass('db.kdbx', password=args.password)       
+pm = PyKeePass('db.kdbx', password="1")  # Hard_Coded Password 
 entry = pm.find_entries(title=args.query,first=True)
 print(entry) 
 print(entry.password)
-- 
2.32.0
